x/concentrated-liquidity/types/params.go

```diff
@@ -0,0 +1,70 @@
+package types
+
+import (
+	fmt "fmt"
+	"strconv"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter store keys.
+var (
+	KeyAuthorizedTickSpacing = []byte("AuthorizedTickSpacing")
+
+	_ paramtypes.ParamSet = &Params{}
+)
+
+// ParamTable for concentrated-liquidity module.
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+func NewParams(authorizedTickSpacing []string) Params {
+	return Params{
+		AuthorizedTickSpacing: authorizedTickSpacing,
+	}
+}
+
+// DefaultParams returns default concentrated-liquidity module parameters.
+func DefaultParams() Params {
+	return Params{
+		AuthorizedTickSpacing: []string{"1", "10", "60", "200"},
```

Comment:
How were these values decided on? Should we have an issue tracking revisiting these?

Reply:
https://github.com/osmosis-labs/osmosis/issues/3684